handlers/server_spec_handler.go

```diff
@@ -19,6 +19,13 @@ type Version struct {
 	ReleaseChannel string `json:"release_channel,omitempty"`
 }
 
+// swagger :route GET /api/server/version SystemAPI idGetSystemVersion
+// Handle GET reqest for system/server version
+//
+//Returns the running Meshery version
+// responses:
+//200: mesheryVersionRespWrapper
```

Comment:
@piyushsingariya @navendu-pottekkat @dhruv0000 are we not giving example responses in docs?

this description doesn’t really tell me the purpose of the endpoint and it’s intended use.

Reply:
we do have samples included in docs. I am not sure why these swagger comments have been edited! @gravishpurohit

Reply:
@piyushsingariya ya, as per the docs I added that and I didn't edit.

Reply:
reset this file `handlers/server_spec_handler.go`, you have edited this, not sure why. This doesn't even have anything related to `/filter` endpoints.